### PR TITLE
py-serpent: update to 1.41, support new pythons

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Update, drop archaic pythons 3.x, support new ones.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Review by smithsp — APPROVED** [review]
I am no longer using MacPorts, so I cannot check these changes, but these changes look fine to me.

**Comment by smithsp** [comment]
Does @barracuda156 want to be the maintainer now?

**Comment by barracuda156** [comment]
@smithsp Thank you. Maybe make the port `nomaintainer` for now? I seldom deal with Python myself; the reason for several updates and new ports for Python from my side in last couple of days is to implement building of one specific port.

File: python/py-serpent/Portfile [hunk 1]
```diff
@@ -15,16 +16,16 @@ license             MIT
 maintainers         {fusion.gat.com:smithsp @smithsp} openmaintainer
```
Comment (smithsp) [inline python/py-serpent/Portfile:16]:
```suggestion
maintainers         nomaintainer
```

**Comment by barracuda156** [comment]
@smithsp Added the suggested commit.

**Pull request merged**